app/Resources/FOSUserBundle/serializer/Model.User.yml

```diff
@@ -0,0 +1,11 @@
+FOS\UserBundle\Model\User:
```

Comment:
@stloyd True, but not related. I just said this file should be placed in `app/Resources/FOSUserBundle/config/serializer/Model.User.yml` IMO. I did so in my project, worked like a charm.